docker-compose.yml

```diff
@@ -9,6 +9,7 @@ web:
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+  command: bundle exec unicorn -l 0.0.0.0:3000
```

Comment:
That's a problem then and means that this won't work with Unicorn on EB. The local `docker-compose.yml` is for orchestrating local containers, and in the `circle/docker-compose-template.yml` is for using compose to orchestrate the testing containers on CircleCI after building the image there. Compose isn't used on EB at all (only the Dockerfile is), so whatever we'll want to accomplish with Docker will need to be specified in the Dockerfile, without the help of compose. I'm going to look at this locally now.